README.md

```diff
@@ -1,2 +1,4 @@
 # WebAutomation_Python2022
-WebDriver Python course
+WebDriver Python course 
+Hello, this is my first repo!
+I am making my first commit in git!
```

Comment:
This would look much cleaner if there were extra blank lines to separate your code.